pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -1382,14 +1386,11 @@ func (pl *dynamicResources) Reserve(ctx context.Context, cs *framework.CycleStat
 		}
 		state.informationsForClaim[index].allocation = allocation
 		state.informationsForClaim[index].allocationDriverName = driverName
-		pl.inFlightAllocations.Store(claim.UID, true)
 		claim = claim.DeepCopy()
 		claim.Status.DriverName = driverName
 		claim.Status.Allocation = allocation
-		if err := pl.claimAssumeCache.Assume(claim); err != nil {
-			return statusError(logger, fmt.Errorf("update claim assume cache: %v", err))
-		}
-		logger.V(5).Info("prepared allocation result", "claim", klog.KObj(claim), "driver", driverName, "allocation", allocation)
+		pl.inFlightAllocations.Store(claim.UID, claim)
+		logger.V(5).Info("prepared allocation result", "claim", klog.KObj(claim), "driver", driverName, "allocation", klog.Format(allocation))
```

Comment:
Is "Reserve" part of the logger name? If not, maybe say reserve somewhere in the message.

Reply:
Will do.